app/src/main/java/com/isc/hermes/model/Searcher.java

```diff
@@ -20,30 +20,87 @@
  * @see CarmenFeature a feature reresented with geojson properties
  */
 public class Searcher {
+
+    CurrentLocationModel currentLocationModel = new CurrentLocationModel();
```

Comment:
Global variables should be initialized in the constructor, unless they are constants.